libs/components/datetime/src/lib/modules/datepicker/fuzzy/datepicker-input-fuzzy.directive.ts

```diff
@@ -578,9 +578,9 @@ export class SkyFuzzyDatepickerInputDirective
 
   /* istanbul ignore next */
   #fuzzyDatesEqual(dateA?: SkyFuzzyDate, dateB?: SkyFuzzyDate): boolean {
-    return (
-      dateA !== undefined &&
-      dateB !== undefined &&
+    return !!(
+      dateA &&
```

Comment:
Angular considers a form control initialized with `undefined` to be `null`, so this check was failing with `null` values before.